decoders/sflow/packet.go

```diff
@@ -1,26 +1,16 @@
 package sflow
 
-import (
-	"fmt"
-	"net/netip"
-)
+import "github.com/netsampler/goflow2/v2/decoders/utils"
```

Comment:
Did `go fmt` remove the parenthesis?

Reply:
I am using `goimports` and yes, it did that.